$DragonFly: src/gnu/usr.bin/cvs/doc/Attic/cvs.texinfo.patch,v 1.1 2006/09/13 19:34:24 corecode Exp $
diff -r 93f8116a881a contrib/cvs-1.12/doc/cvs.texinfo
--- a/contrib/cvs-1.12/doc/cvs.texinfo	Sat Sep 09 18:10:13 2006 +0200
+++ b/contrib/cvs-1.12/doc/cvs.texinfo	Wed Sep 13 17:10:27 2006 +0200
@@ -9144,7 +9144,9 @@ Changes: nothing.
 
 For each file in @var{files}, print the head revision
 of the trunk, together with information on the last
-modification for each line.  
+modification for each line.  If @samp{-b} is specified,
+the first modification after the revision selected by
+@samp{-r} is shown.  
 
 @menu
 * annotate options::            annotate options
@@ -9160,6 +9162,9 @@ them):
 them):
 
 @table @code
+@item -b
+Backwards, show when a line was removed.
+
 @item -l
 Local directory only, no recursion.
 
